### ISSUE TITLE
Export to USFM3 is broken

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#4631]
0.10.0 (a48d474) 
No idea which issue broke this.
Import https://git.door43.org/katalakay/en_1th
Open in wA and see that it is mostly aligned
Export to USFM3
This is the exported file:
[53-1TH.usfm.zip](https://github.com/unfoldingWord-dev/translationCore/files/2116244/53-1TH.usfm.zip)

**Comment by RoyalSix** [comment]
@cckozie I am not able to replicate this on develop

**Comment by RoyalSix** [comment]
## Notes:
- Not sure if this is the main problem but from investigating this issue I found that an improperly parsed verse string that includes a footnote doesn't match the corresponding alignment words and therefore is causing a false positive "invalidated alignments to show"
i.e.
verse string:`"and we sent Timothy, our brother and fellow worker for God in the gospel of Christ, to strengthen and comfort you regarding your faith,
\f + \ft Some other versions read, \fqa We sent Timothy, our brother and God's servant in the gospel of Christ \fqa* . \f*"`
gets parsed as: 
```
[{"tag":"w","type":"word","text":"and","occurrence":1,"occurrences":3},{"tag":"w","type":"word","text":"we","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"sent","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"Timothy","occurrence":1,"occurrences":1},{"type":"text","text":","},{"tag":"w","type":"word","text":"our","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"brother","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"and","occurrence":2,"occurrences":3},{"tag":"w","type":"word","text":"fellow","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"worker","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"for","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"God","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"in","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"the","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"gospel","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"of","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"Christ","occurrence":1,"occurrences":1},{"type":"text","text":","},{"tag":"w","type":"word","text":"to","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"strengthen","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"and","occurrence":3,"occurrences":3},{"tag":"w","type":"word","text":"comfort","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"you","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"regarding","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"your","occurrence":1,"occurrences":1},{"tag":"w","type":"word","text":"faith","occurrence":1,"occurrences":1},{"type":"text","text":","},{"tag":"f","type":"footnote","content":"+ \\ft Some other versions read, \\fqa We sent Timothy, our brother and God's servant in the gospel of Christ \\fqa* . "}]
```

which doesn't match alignments:
```
[{"topWords":[{"word":"καὶ","strong":"G25320","lemma":"καί","morph":"Gr,CC,,,,,,,,","occurrence":1,"occurrences":3}],"bottomWords":[{"word":"and","occurrence":1,"occurrences":4,"type":"bottomWord"}]},{"topWords":[{"word":"ἐπέμψαμεν","strong":"G39920","lemma":"πέμπω","morph":"Gr,V,IAA1,,P,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"we","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"Τιμόθεον","strong":"G50950","lemma":"Τιμόθεος","morph":"Gr,N,,,,,AMS,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"τὸν","strong":"G35880","lemma":"ὁ","morph":"Gr,EA,,,,AMS,","occurrence":1,"occurrences":1},{"word":"ἀδελφὸν","strong":"G00800","lemma":"ἀδελφός","morph":"Gr,N,,,,,AMS,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"ἡμῶν","strong":"G14730","lemma":"ἐγώ","morph":"Gr,RP,,,1G,P,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"καὶ","strong":"G25320","lemma":"καί","morph":"Gr,CC,,,,,,,,","occurrence":2,"occurrences":3}],"bottomWords":[]},{"topWords":[{"word":"διάκονον","strong":"G12490","lemma":"διάκονος","morph":"Gr,N,,,,,AMS,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"fellow","occurrence":1,"occurrences":1,"type":"bottomWord"},{"word":"worker","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"τοῦ","strong":"G35880","lemma":"ὁ","morph":"Gr,EA,,,,GMS,","occurrence":1,"occurrences":2},{"word":"Θεοῦ","strong":"G23160","lemma":"θεός","morph":"Gr,N,,,,,GMS,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"for","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"ἐν","strong":"G17220","lemma":"ἐν","morph":"Gr,P,,,,,D,,,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"τῷ","strong":"G35880","lemma":"ὁ","morph":"Gr,EA,,,,DNS,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"εὐαγγελίῳ","strong":"G20980","lemma":"εὐαγγέλιον","morph":"Gr,N,,,,,DNS,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"τοῦ","strong":"G35880","lemma":"ὁ","morph":"Gr,EA,,,,GMS,","occurrence":2,"occurrences":2},{"word":"Χριστοῦ","strong":"G55470","lemma":"χριστός","morph":"Gr,N,,,,,GMS,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"εἰς","strong":"G15190","lemma":"εἰς","morph":"Gr,P,,,,,A,,,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"τὸ","strong":"G35880","lemma":"ὁ","morph":"Gr,RD,,,,ANS,","occurrence":1,"occurrences":1}],"bottomWords":[]},{"topWords":[{"word":"στηρίξαι","strong":"G47410","lemma":"στηρίζω","morph":"Gr,V,NAA,,,,,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"to","occurrence":1,"occurrences":1,"type":"bottomWord"},{"word":"strengthen","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"ὑμᾶς","strong":"G47710","lemma":"σύ","morph":"Gr,RP,,,2A,P,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"you","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"καὶ","strong":"G25320","lemma":"καί","morph":"Gr,CC,,,,,,,,","occurrence":3,"occurrences":3}],"bottomWords":[]},{"topWords":[{"word":"παρακαλέσαι","strong":"G38700","lemma":"παρακαλέω","morph":"Gr,V,NAA,,,,,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"comfort","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"ὑπὲρ","strong":"G52280","lemma":"ὑπέρ","morph":"Gr,P,,,,,G,,,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"regarding","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"τῆς","strong":"G35880","lemma":"ὁ","morph":"Gr,EA,,,,GFS,","occurrence":1,"occurrences":1},{"word":"πίστεως","strong":"G41020","lemma":"πίστις","morph":"Gr,N,,,,,GFS,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"faith","occurrence":1,"occurrences":1,"type":"bottomWord"}]},{"topWords":[{"word":"ὑμῶν","strong":"G47710","lemma":"σύ","morph":"Gr,RP,,,2G,P,","occurrence":1,"occurrences":1}],"bottomWords":[{"word":"your","occurrence":1,"occurrences":1,"type":"bottomWord"}]}]
```

**Comment by cckozie** [comment]
Fixed in 0.10.1 (f43aa52)
